frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import * as _ from 'lodash';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector, useDispatch } from 'react-redux';
+import { match as RMatch } from 'react-router-dom';
+import { Table, TableHeader, TableBody, SortByDirection } from '@patternfly/react-table';
+import { FilterToolbar } from '@console/internal/components/filter-toolbar';
+import { getURLSearchParams } from '@console/internal/components/utils';
+import { getAlertsAndRules } from '@console/internal/components/monitoring/utils';
+import { monitoringSetRules } from '@console/internal/actions/ui';
+import { useURLPoll } from '@console/internal/components/utils/url-poll-hook';
+import { PrometheusRulesResponse } from '@console/internal/components/monitoring/types';
+import { RootState } from '@console/internal/redux';
+import {
+  alertSeverityOrder,
+  alertStateOrder,
+  alertingRuleStateOrder,
+} from '@console/internal/reducers/monitoring';
+import {
+  monitoringAlertRows,
+  monitoringAlertColumn,
+  alertFilters,
+} from './monitoring-alerts-utils';
+import './MonitoringAlerts.scss';
+
+type MonitoringAlertsProps = {
+  match: RMatch<{
+    ns?: string;
+  }>;
+};
+
+const POLL_DELAY = 15 * 1000;
+
+const MonitoringAlerts: React.FC<MonitoringAlertsProps> = ({ match }) => {
+  const {
+    'rowFilter-alert-state': alertStateParams,
+    'rowFilter-alert-severity': alertSeverityParams,
+    name,
+  } = getURLSearchParams();
+  const [sortBy, setSortBy] = React.useState({});
+  const [rows, setRows] = React.useState([]);
+  const [collapsedRows, setCollapsedRows] = React.useState([]);
+  const dispatch = useDispatch();
+  const rules = useSelector((state: RootState) => state.UI.getIn(['monitoring', 'devRules']));
+  const namespace = match.params.ns;
+  const { prometheusTenancyBaseURL } = window.SERVER_FLAGS;
```

Comment:
Should probably import `PROMETHEUS_TENANCY_BASE_PATH` instead of using `window.SERVER_FLAGS` to be consistent with how we're generally doing this elsewhere.